src/Workspaces/VisualBasic/Portable/LanguageServices/VisualBasicSemanticFactsService.vb

```diff
@@ -290,5 +293,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
             Return {semanticModel.GetDeclaredSymbol(memberDeclaration, cancellationToken)}
         End Function
+
+        Private Function ISemanticFactsService_GenerateUniqueName(semanticModel As SemanticModel, location As SyntaxNode, containerOpt As SyntaxNode, baseName As String, cancellationToken As CancellationToken) As SyntaxToken Implements ISemanticFactsService.GenerateUniqueName
```

Comment:
line way too long

Reply:
in this type, considering so many methods are simply to just represent the interface signature, we don't wrap and just forward things along.  I agree it's not consistent, but it helps keep this type from literally becoming 5x as long and having the signatures mask what's actually relevant.